packages/web/src/shared/Link.tsx

```diff
@@ -3,14 +3,19 @@ import * as React from 'react'
 
 interface Props {
   href: string
-  children: any
+  children: React.ReactNode
   prefetch?: boolean
+  passHref?: boolean
 }
 
-export default ({ href, children, prefetch = false }: Props) => {
-  return (
-    <Link prefetch={prefetch} href={href}>
-      {children}
-    </Link>
-  )
+export default ({ href, children, passHref, prefetch = false }: Props) => {
+  if (href) {
+    return (
+      <Link prefetch={prefetch} href={href} passHref={passHref}>
+        {children}
+      </Link>
+    )
+  } else {
+    return <>{children}</>
```

Comment:
this is so Link is not rendered when we are just setting a `onPress` as rendering without a `href` caused issues that the tests picked up